es-models/case_centric/case_centric.mapping.yaml

```diff
@@ -46,6 +52,12 @@ properties:
         type: keyword
       updated_datetime:
         type: date
+      vital_status:
```

Comment:
If this is a new field, I think this should just type keyword:
      vital_status:
        type: keyword